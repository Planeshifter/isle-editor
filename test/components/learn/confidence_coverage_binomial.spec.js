// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import LearnConfidenceCoverageBinomial from '@isle-project/components/learn/confidence-coverage-binomial/main.js';


// TESTS //

describe( '<LearnConfidenceCoverageBinomial />', function test() {
	it( 'renders a learning component', () => {
		const learning = renderer.create( <LearnConfidenceCoverageBinomial /> );
		expect( learning ).toMatchSnapshot();
	});
});
