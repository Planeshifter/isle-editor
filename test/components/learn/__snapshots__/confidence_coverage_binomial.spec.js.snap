// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LearnConfidenceCoverageBinomial /> renders a learning component 1`] = `
<div
  className="coverage-card card"
>
  <h4
    className="card-header"
  >
    confidence-interval-coverage
  </h4>
  <div
    className="card-body"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div>
          <p>
            Now we will switch to asking a Yes/No question about a population. We are interested in estimating the true population proportion 
            <div />
             of "Yes" answers (for example, what proportion of the population has blue eyes?).  We can take a sample of size 
            <div />
            , find how many observations in our sample are a "Yes" (X), and then estimate the true proportion 
            <div />
             with 
            <div />
            . Then 
            <div />
            . Our confidence interval is then 
            .
          </p>
          <p>
            For our choice of sample size (n), true proportion 
            <div />
            , and confidence level, we will simulate 
            <div />
             different samples from our normal distribution and calculate the corresponding sample proportions and confidence intervals.
          </p>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-md-5"
        />
        <div
          className="col-md-7"
        >
          <div
            className="card"
          >
            <h4
              className="card-header"
            >
              confidence-intervals
            </h4>
            <div
              className="card-body"
            >
              <p>
                capture-true-proportion
                 
                <b>
                  (
                  coverage
                  : 
                  0
                  ).
                </b>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
